Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -497,7 +518,7 @@ public void indexConcepts(String userName)
         if (operator == null || operator.equalsIgnoreCase(SearchParamters.OP_AND)) {
             occur = BooleanClause.Occur.MUST;
         }
-
+        try {
```

Comment:
where is the catch block for this try? Give that I can't see it, it probably means the try block is waayyyy to big. Catch exceptions where they occur and deal with them appropriately right where they are thrown.

Reply:
Yes, it is because the try block is big. Will handle this.